### PR TITLE
bump(main/plutolang): 0.9.0

### PR DESCRIPTION
**Pull request by cattokomo**

### CODE REVIEW

**Review by TomJo2000 — APPROVED** [review]

File: packages/plutolang/build.sh [hunk 1]
```diff
@@ -2,11 +2,11 @@ TERMUX_PKG_HOMEPAGE=https://plutolang.github.io/
 TERMUX_PKG_DESCRIPTION="A superset of Lua 5.4, with unique features, optimizations, and improvements"
 TERMUX_PKG_LICENSE="MIT"
 TERMUX_PKG_MAINTAINER="@termux"
```
Comment (TomJo2000) [inline packages/plutolang/build.sh:4]: Would you mind setting yourself as the maintainer for this package?

Comment (cattokomo) [reply packages/plutolang/build.sh]: Resolved in [9b2b93e](https://github.com/termux/termux-packages/pull/19695/commits/9b2b93eb2ed0593d34253e841ad0f74c6db51972).

Comment (TomJo2000) [reply packages/plutolang/build.sh]: Will still need to be squashed.
We *can* do that during Merge, but usually we prefer the author to do this in order to ensure everything is being committed as intended.

You can squash the commits using `git rebase -i HEAD~2`, and selecting `s`quash for the second commit to merge it into the first one.

Comment (cattokomo) [reply packages/plutolang/build.sh]: The commits are now squashed.

Comment (TomJo2000) [reply packages/plutolang/build.sh]: Thank you.
Is this ready to merge then, or are you still testing it?

Comment (cattokomo) [reply packages/plutolang/build.sh]: I still need to test `-fstack-protector-strong` on the Pluto `testes/` directory, I'll inform you once I'm done.

Comment (cattokomo) [reply packages/plutolang/build.sh]: Done tested it, you may merge this now.

**Comment by TomJo2000** [comment]
Alright, will merge around 8pm CET (2 hours, 15 minutes from now) if nothing else comes up.

**Pull request merged**